make-debs.sh

```diff
@@ -93,7 +93,9 @@ Suite: stable
 Components: main
 Architectures: i386 amd64 source
 EOF
-ln -s $codename/conf conf
+if [ ! -e conf ]; then
+    ln -s $codename/conf conf
```

Comment:
It's not the same. -f is for force. the problem is the target "conf" exists and it's a directory so ln creates a symlink inside conf and then I get conf/conf.

it's more like "ln -sT $codename/conf conf" but then ln will return an error as conf exists and the bash will fail as it runs with -e